src/constants.ts

```diff
@@ -140,9 +142,11 @@ export const enum TextureFormat {
   RGBA32Uint = 'rgba32uint',
   RGBA32Sint = 'rgba32sint',
   RGBA32Float = 'rgba32float',
+  Depth16Unorm = 'depth16unorm',
   Depth32Float = 'depth32float',
   Depth24Plus = 'depth24plus',
   Depth24PlusStencil8 = 'depth24plus-stencil8',
+  Stencil8 = 'stencil8',
```

Comment:
It seems that `stencil8` is missing here. In addition, all bc formats are not added into the list, because they are extensions? I didn't add depth24unorm-stencil8 and depth32float-stencil8, either.

Reply:
They're only omitted because the types are out of date wrt the spec and no one has ported this particular change forward.

Reply:
This particular file is probably not used by anyone anymore (cts moved from using this to using `as const`, and I think(?) babylon maintains its own copy), so it hasn't been kept as up to date.